system/src/ble_control_request_channel.cpp

```diff
@@ -1044,6 +1044,11 @@ int BleControlRequestChannel::initProfile() {
         LOG_DEBUG(TRACE, "BLE profile already initialized");
         return SYSTEM_ERROR_INVALID_STATE;
     }
+#if HAL_PLATFORM_RTL872X
+    SCOPE_GUARD ({
+        hal_ble_stack_deinit(nullptr);
```

Comment:
What if `BLE.on()` was called previously by user app? We should probably check the state before this was called?

Reply:
Fixed

Reply:
Fixed